### PR TITLE
fix #40 RHEL/Centos: manage icinga2-selinux package

### LINKED ISSUE #40: RHEL/Centos: manage icinga2-selinux package
I hit [this](https://github.com/Icinga/icinga2/issues/7832) problem after the first `icinga2` role run. I resolved it by installing the `icinga2-selinux` package. I recommend installing `icinga2-selinux` by default in the same dnf transaction as the `icinga2` package.
Note that installing the `icinga2-selinux` as a pre-task is not an option as it has a dependency on the base `icinga2` package.
```
# rpm -qR icinga2-selinux | grep el8\.icinga
icinga2-bin = 2.12.3-1.el8.icinga
```

### PR DESCRIPTION
**Pull request by lbetz**

### CODE REVIEW

File: roles/icinga2/tasks/install_on_RedHat.yml [hunk 1]
```diff
@@ -2,3 +2,8 @@
   yum:
     name: icinga2
     state: present
+- name: Yum - install package icinga2-selinux
+  yum:
+    name: icinga2-selinux
+    state: present
+  when: ansible_selinux.status == "enabled"
```
Comment (lazyfrosch) [inline roles/icinga2/tasks/install_on_RedHat.yml:9]:
```suggestion
  when: ansible_selinux is defined and ansible_selinux.status == "enabled"
```

**Review by lazyfrosch — CHANGES_REQUESTED** [review]
Good idea, I would check if the ansible_selinux fact is even present

**Comment by mkayontour** [comment]
This is right, to avoid the failure of the role when the ansible_selinux.state is missing. @lbetz are you happy with this?

**Comment by lbetz** [comment]
@mkayontour sure!

**Pull request merged**